schemas/serialization.schema.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Doesn't this already exist somewhere?

Reply:
Yeah, it does: I'll see that I can remove it.